CMakeLists.txt

```diff
@@ -366,3 +366,10 @@ if(AUTOPEP8_FOUND)
   autopep8_add_subdirectory("${CMAKE_SOURCE_DIR}/bindings" "")
   autopep8_add_subdirectory("${CMAKE_SOURCE_DIR}/scripts" "")
 endif()
+
+# Removes every file in the build folder except the external libraries to
+# prevent redownloading external libraries
+add_custom_target(
+    deepclean
+    COMMAND python ${CMAKE_SOURCE_DIR}/scripts/developer_utils.py deepclean
```

Comment:
Does the script work with Python 2? It might need a few `from __future__ import ...`

Reply:
yes it does